pkg/helper/download/download.go

```diff
@@ -8,7 +8,7 @@ import (
 	"runtime"
 	"strings"
 
-	"github.com/dustin/go-humanize"
+	humanize "github.com/dustin/go-humanize"
```

Comment:
Unrelated change

Reply:
Let's keep this. I'm willing to bet we will be fighting the bots on every PR and visit if we don't. It'll keep trying to name these as various parsers read/sort/rewrite things. For some of these random libraries, it's likely doing this to give it maximum freedom and clarity in the namespace.